generative-ai-overview/generative-ai-overview.md

```diff
@@ -25,8 +26,8 @@ What are some areas of strength and weakness of LLMs?
 * [5 Practical Business Use Cases for Large Language Models](https://opendatascience.com/5-practical-business-use-cases-for-large-language-models/) -  From opendatascience.com
 
 How does bias impact LLMs?
-* *Content Warning: there are no graphic depictions, but there are examples discussed of harmful sexist and racist content created by generative AIs.* [Exploration of bias in ChatGPT and other AI models](https://www.insider.com/chatgpt-is-like-many-other-ai-models-rife-with-bias-2023-1) - from Insider.com
-* [Depression in Black people goes unnoticed by AI models analyzing language in social media posts](https://www.pennmedicine.org/news/news-releases/2024/march/depression-in-black-people-unnoticed-by-ai-analyzing-social-media) - from Penn Medicine News
+* *Content Warning: there are no graphic depictions, but there are examples discussed of harmful racist content created by generative AIs.* [Covert Racism in AI: How Language Models Are Reinforcing Outdated Stereotypes](https://hai.stanford.edu/news/covert-racism-ai-how-language-models-are-reinforcing-outdated-stereotypes) - from Stanford University
```

Comment:
What if we move the link to it's own nested bullet? Having it together with the content warning is harder to distinguish for me.

Also, I suggest dropping "discussed" just from a sentence flow perspective, or if the preference is to keep it, switch "examples discussed of" to "examples discussed about".

```suggestion
* *Content Warning: there are no graphic depictions, but there are examples of harmful racist content created by generative AIs.*
  * [Covert Racism in AI: How Language Models Are Reinforcing Outdated Stereotypes](https://hai.stanford.edu/news/covert-racism-ai-how-language-models-are-reinforcing-outdated-stereotypes) - from Stanford University
```